client/my-sites/design-preview/index.js

```diff
@@ -220,20 +221,20 @@ function mapStateToProps( state ) {
 		};
 	}
 
+	const previewSlug = getPreviewSlug( state );
 	const { previewMarkup, customizations, isUnsaved } = state.preview[ selectedSiteId ];
 	return {
 		selectedSite,
 		selectedSiteId,
+		previewSlug,
```

Comment:
I guess it's consistent, but seems a bit excessive to assign the variable separately just for object shorthand notation, when could be inlined:

```js
previewSlug: getPreviewSlug( state )
```